uploadcare.js

```diff
@@ -49,6 +49,10 @@ if (!RedactorPlugins) var RedactorPlugins = {};
                         } else {
                             $this.insert.html('<a href="' + this.cdnUrl + '">' + info.name + '"</a>', false);
                         }
+
+                        if ($.isFunction($opts.uploadCallback)) {
```

Comment:
`uploadCompleteCallback` seems to be more precise name for this.
what do you think?